ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -436,7 +436,7 @@
 				CODE_SIGN_IDENTITY = "Apple Development";
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 18;
-				DEVELOPMENT_TEAM = 9CX5MA4DG5;
+				DEVELOPMENT_TEAM = 53K3QSV7XZ;
```

Comment:
Please don't change dev team, thanks. I added you to the Hypha dev team so should not be needed

But in case XCode acts up, just don't check it in

Reply:
Sorry it was change by mistake.